enftun/README.md

```diff
@@ -20,10 +20,18 @@ FROM xaptum:enftun
 
 ### Test Shell
 
-To run this image in a local container, run:
-```
-make shell
-```
+To run this image in a local container:
+- Create a subdirectory named `enf0`
+    ``` 
+    mkdir enf0
+    ```
+- Copy the key files generated using the  [enf-services
```

Comment:
To be more explicit, rather than `key files` how about `the private key and certificate`.